source/components/Select/index.jsx

```diff
@@ -0,0 +1,200 @@
+// @Flow
+/* eslint no-lonely-if: 0 */
+/* eslint jsx-a11y/click-events-have-key-events: 0 */
+/* eslint jsx-a11y/no-static-element-interactions: 0 */
+
+// disabled due to prettierrc:
+/* eslint operator-linebreak: 0 */
+/* eslint object-curly-newline: 0 */
+
+import * as React from 'react';
+
+import dropDownArrow from 'assets/drop_down_arrow.svg';
+import './styles.scss';
+
+type Props = {
+  label: string,
+  tip: string,
+  items: Array<{ text: string }>,
```

Comment:
Let's just make this a string array

Reply:
I made it an array of objects so each selection can hold more data than just the string it displays.  This is how the material-ui select component works.  Each object can optionally hold another `value` attribute in the case that you are working with data that isn't strings.  If you don't think that's necessary I can change this and the `onSelect` function mentioned above.

Reply:
Ok, I don't see a use case where we would need this but if you think we should leave it, then it's fine.